server/stats/event_service.py

```diff
@@ -57,13 +57,25 @@ def __init__(self, api_accessor: ApiAccessor):
                 "event_id": string,
                 "count": long
             }]
+        Else, returns None
         """
         self._logger.debug("Recording %d events", len(queue))
-        data = dict(updates=queue)
-        response, content = await self.api_accessor.api_post("/events/recordMultiple", player_id, data=data)
+        response, content = await self.api_accessor.update_events(queue, player_id)
 
-        return json.loads(content.decode('utf-8'))['updated_events']
+        if response < 300:
+            """
+            Converting the Java API data to the structure mentioned above
+            """
+            events_data = []
+            for event in json.loads(content)['data']:
+                converted_event = dict(
+                    event_id=event['attributes']['eventId'],
+                    count=event['attributes']['count']
+                )
+                events_data.append(converted_event)
 
+            return events_data
+        return None
```

Comment:
It's better to not return a empty events_data with the wrong structure because in case of event service it will be fine (the returned value is not used anyway), but in case of achievement service it could cause trouble in the server OR send wrong data to the client which we don't want. In case there's an issue with the API, i think we are better returning None and wait for the storm to pass.

Reply:
Make code simple as much as it possible. Let it return just list. If return value is not used - don't return it.

Reply:
Why should I return data with wrong structure on failure and data with right structure on success ? This is not as simple as simply returning None